COLLADAMaya/include/COLLADAMayaSyntax.h

```diff
@@ -38,10 +38,13 @@ namespace COLLADAMaya
     static const char* PROFILE_COLLADA                              = "COLLADA";
     static const char* PROFILE_MAYA                                 = "OpenCOLLADAMaya";
     static const char* PROFILE_MAX                                  = "OpenCOLLADA3dsMax";
+	static const char* PROFILE_FL4RE								= "fl4re";
```

Comment:
Sorry i have just seen you made a comment ...
I'm actually updating the physic exporter so now no need to use PROFILE_FL4RE anymore.